src/main/java/seedu/duke/Parser.java

```diff
@@ -63,7 +63,7 @@ public String processCommand(String command, String[] inputs)
         }
     }
 
-    private static boolean isNumeric(String strNum) {
+    private boolean isNumeric(String strNum) {
         if (strNum == null) {
```

Comment:
this logic is a bit confused because the false return value means the parameter is not numeric only. However, it seems it could be false when the strNum is null as well.